packages/perseus/src/widgets/sorter/sorter-validator.ts

```diff
@@ -0,0 +1,17 @@
+import _ from "underscore";
+
+import type {Rubric, UserInput} from "./sorter.types";
+import type {PerseusScore} from "../../types";
+
+function sorterValidator(userInput: UserInput, rubric: Rubric): PerseusScore {
+    const correct = _.isEqual(userInput.values, rubric.correct);
```

Comment:
Would love to remove this `_.isEqual`, but didn't know of or find a quick way to do so.

Reply:
It looks like our `deepEq()` function in https://github.com/Khan/perseus/blob/12b8e01bf89abbc88d6bfc9bf243d5a0e95b5ed3/packages/perseus/src/util.ts#L547 would do what we need.

Reply:
Wow! Thank you so much! Below is how I used it, and all tests pass!
```
import Util from "../../util";
...
const correct = Util.deepEq(userInput.values, rubric.correct);
```